### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by c0derMo**

### CODE REVIEW

**Review by a-bumblebee — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,36 +1,50 @@
...
-- `npm run start`: Starts the built project
-- `npm run lint`: Runs eslint on the entire sourcecode (also done precommit)
-- `npm run prettier`: Runs prettier on the entire sourcecode (also done precommit)
+A web app for creating learning sequences.
+
+## Setup
+Setting up the platform is as easy as it gets:
+1. Obtain a built version of the platform
+    1. Either download a prebuilt release
+    2. Alternatively, build from source
```
Comment (Kr0nox) [inline README.md]: how?

File: README.md [hunk 2]
```diff
@@ -1,36 +1,50 @@
...
+
+`.env` files can also be used to set these variables.
+
+## Adding a custom imprint
+The imprint file is located inside the `dist/imprint.html`. The content of this file can be changed to your liking to add your custom imprint.
+
+Alternatively, the imprint can be changed in the source files at `frontend/public/imprint.html`, after which the platform needs to be recompiled.
+
+## Security warning - automatic file download
+The platform automatically downloads & served the required H5P libraries.
```
Comment (Kr0nox) [inline README.md]: serves not served

File: README.md (hunk 1 above)
Comment (Kr0nox) [reply README.md]: maybe a link to the bottom section

File: backend/h5p/H5PServer.ts [hunk 3]
```diff
@@ -71,6 +71,16 @@ export class H5PServer {
             config
         );
 
+        // Clean up temporary files every 5 minutes
```
Comment (Kr0nox) [inline backend/h5p/H5PServer.ts]: maybe a comment that this is required/suggested by the h5p library?

**Review by Kr0nox — CHANGES_REQUESTED** [review]

**Review by Kr0nox — APPROVED** [review]

**Pull request merged**